requirements/tests.txt

```diff
@@ -7,23 +7,15 @@
 #
 attrs==22.2.0
     # via pytest
-exceptiongroup==1.1.0
-    # via pytest
-importlib-metadata==6.0.0
-    # via
-    #   pluggy
-    #   pytest
 iniconfig==2.0.0
     # via pytest
 packaging==23.0
     # via pytest
 pluggy==1.0.0
     # via pytest
 pytest==7.2.1
+    # via
+    #   -r requirements/tests.in
+    #   pytest-asyncio
+pytest-asyncio==0.20.3
     # via -r requirements/tests.in
-tomli==2.0.1
```

Comment:
Please relock on python 3.7 so all the dependencies of `pytest` are locked.

Reply:
Done